### DISCUSSION TITLE
使用 npx create-dumi 创建报错 error Couldn't find package "@commitlint/cli@^17.1.2" required by "components@0.0.1" on the "npm" registry.

### DISCUSSION THREAD

**Discussion by jackason** [umijs/dumi#2021] [Q&A]
直接使用 npx create-dumi 创建  报错 error Couldn't find package "@commitlint/cli@^17.1.2" required by "components@0.0.1" on the "npm" registry.

dumi版本  2.2.13
操作系统版本  window11
Node.js 版本  16.15.1
yarn版本 1.22.19

社区大佬们。这个怎么解决呀

**Comment by PeachScript** [comment]
这是依赖安装报错，NPM 官方源上有这个版本：https://www.npmjs.com/package/@commitlint/cli?activeTab=versions

所以需要自行检查使用的 NPM 源是否完整同步了官方源
